src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -216,6 +216,72 @@ elementFormDefault="qualified">
               </xs:documentation>
             </xs:annotation>
           </xs:attribute>
+          <xs:attribute name="IncludeAssets" type="xs:string" use="optional">
+            <xs:annotation>
+              <xs:documentation>
+                <!-- _locID_text="PackageReference_Attribute_IncludeAssets" _locComment="" -->Assets to include from this reference
+              </xs:documentation>
+            </xs:annotation>
+          </xs:attribute>
+          <xs:attribute name="ExcludeAssets" type="xs:string" use="optional">
+            <xs:annotation>
+              <xs:documentation>
+                <!-- _locID_text="PackageReference_Attribute_ExcludeAssets" _locComment="" -->Assets to exclude from this reference
+              </xs:documentation>
+            </xs:annotation>
+          </xs:attribute>
+          <xs:attribute name="PrivateAssets" type="xs:string" use="optional">
+            <xs:annotation>
+              <xs:documentation>
+                <!-- _locID_text="PackageReference_Attribute_PrivateAssets" _locComment="" -->Assets that are private in this reference
+              </xs:documentation>
+            </xs:annotation>
+          </xs:attribute>
+        </xs:extension>
+      </xs:complexContent>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="FrameworkReference" substitutionGroup="msb:Item">
+    <xs:annotation>
+      <xs:documentation>
+        <!-- _locID_text="FrameworkReference" _locComment="" -->Reference to a shared framework.
```

Comment:
For consistency, drop trailing periods on all of these new ones.

Reply:
Well, this is super inconsistent already. There are a bunch that have periods and a bunch without. If we are going to make this consistent, I would rather make it consistent across the board. Probably by adding periods in the places without it.

@rainersigwald what do you think?